fdbclient/admin_client.go

```diff
@@ -628,7 +628,7 @@ func (client *cliAdminClient) KillProcesses(addresses []fdbv1beta2.ProcessAddres
 	}
 
 	killCommand := fmt.Sprintf(
-		"kill; kill %[1]s; sleep 1; kill %[1]s; sleep 1; kill %[1]s",
+		"kill; kill %[1]s; sleep 1; kill %[1]s; sleep 5; status",
```

Comment:
```suggestion
		"kill; kill %[1]s; sleep 1; kill %[1]s; sleep 5",
```

Only the sleep should be fine.